app/assets/stylesheets/application/atoms/_logo.scss

```diff
@@ -1,9 +1,8 @@
-%logo,
 .logo {
+  display: inline-block;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets